src/transformState.js

```diff
@@ -4,8 +4,39 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
+function modifyState(state, actions) {
+  for (const action of actions) {
+    const { type } = action;
+
+    switch (type) {
+      case 'addProperties':
+        const { extraData } = action;
+
+        Object.assign(state, extraData);
```

Comment:
it isn't necessary to declare const { extraData } 
it would be better to assign it right away Object.assign(state, action.extraData);
fix it everywhere

Reply:
I don't understand why i can't push changes
![Screenshot_38](https://github.com/mate-academy/js_stateful-object/assets/123593139/658c3b5e-b75f-4064-8464-3f051e4302d1)